cis_env.py

```diff
@@ -14,18 +15,56 @@ def curl(pos, vec_len=0.004, coor_origin=conf.CURL_CENTRE):
     return curl_pos * vec_len
 
 
-def feed_cell(cell, cell_dict, already_averaged_dict, food_factor=1):
-    f = random.uniform(0, 1)
-    if f < dna_decoding.food_theshold(cell.dna):
-        cell.energy_level += int(dna_decoding.food_amount(cell.dna)
-                                 * food_factor)
-    if conf.GENERAL_ENERGY_CONSUMPTION < cell.energy_level:
-        cell.energy_level -= conf.GENERAL_ENERGY_CONSUMPTION
-    else:
-        cell.energy_level = 0
+def feed(cell, time_step):
+    """
+        Give the cell food,
+        depending on position and time_step.
+    """
 
-    average_out_energy_in_connected_cells(
-        cell, cell_dict, already_averaged_dict)
+    #  get value from the food_function --> definition area: {-3, 3}
+    food_value = food_function(cell.pos, time_step)
+
+    # normalize food_value to {0, 1}
+    food_value = normalize(food_value)
+
+    # check if cell gets (the fixed amount of) food
+    if food_value < conf.FOOD_THRESHOLD:
+        cell.energy_level += conf.FOOD_ENERGY
+
+    return cell
+
+
+def normalize(num, definition_area_size=6):
```

Comment:
The function name `normalize` is too brought, normalization can reference to wide range of procedures and processes, it isn't clear which one of them you reference.